sdk/servicebus/Azure.Messaging.ServiceBus/samples/DeadLetterQueue/DeadLetterQueue.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net5.0</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Azure.Identity" Version="1.2.1" />
+      <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.2.0-beta.2" />
```

Comment:
Should we comment this out until the project reference is removed?   _(to be honest, I'm surprised this builds.... I didn't realize that the references would be normalized.)_

Reply:
I think project reference will override package reference, but yes I can comment this out.